server/repository/migrations/postgres/2023-03-09-194728_v1_1_5/up.sql

```diff
@@ -0,0 +1,43 @@
+ALTER TABLE name ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE changelog ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE clinician ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE clinician_store_join ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE name_store_join ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE document ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+
+/* View wasn't being dropped in down.sql, so have moved it here to recreate view */
+DROP VIEW IF EXISTS changelog_deduped;
+CREATE VIEW changelog_deduped AS
+    SELECT t1.cursor,
+        t1.table_name,
+        t1.record_id,
+        t1.row_action,
+        t1.name_id,
+        t1.store_id,
+        t1.is_sync_update
+    FROM changelog t1
+    WHERE t1.cursor = (SELECT max(t2.cursor) 
+                    from changelog t2
+                    where t2.record_id = t1.record_id)
+    ORDER BY t1.cursor;
+
+CREATE OR REPLACE FUNCTION update_changelog()
+RETURNS trigger AS
+$$
+     DECLARE
+     BEGIN
+        IF (TG_OP = 'DELETE') THEN
+            INSERT INTO changelog (table_name, record_id, row_action, is_sync_update)
+              VALUES (TG_TABLE_NAME::changelog_table_name, OLD.id, 'DELETE', OLD.is_sync_update);
```

Comment:
this should only be done in name and clinician, method is common across most postgres trigger, and is_sync_update is not present (try doing postgres tests, most tests would fail with a message that's not straight away apparent, I had a deep look into it and it was the trigger). 

i.e. see how: `repository/migrations/postgres/2022-02-23T11-30_create_invoice_trigger/up.sql` is different from say, repository/migrations/postgres/2022-02-23T10-30_create_stock_line_trigger/up.sql

Reply:
[e8ab173](https://github.com/openmsupply/open-msupply/pull/1338/commits/e8ab17322acb36e12bb85c784fbd265717528968)
🙏🏻